app/src/main/java/org/eyeseetea/malariacare/layout/adapters/dashboard/ReviewScreenAdapter.java

```diff
@@ -112,7 +117,13 @@ public View getView(final int position, View convertView, ViewGroup parent) {
 
         TableRow rowView = (TableRow) this.lInflater.inflate(getRecordLayout(), parent, false);
 
-        ReviewFragmentStrategy reviewFragmentStrategy = new ReviewFragmentStrategy();
+        ReviewFragmentStrategy reviewFragmentStrategy =
```

Comment:
Really ReviewFragmentStrategy is not a strategy for ReviewFragment is a strategy for ReviewScreenAdapter, can you rename to ReviewScreenAdapterStrategy